index.html

```diff
@@ -47,11 +47,11 @@ <h2 class="intro-title">I’am a Software Developer</h2>
        </div>
```

Comment:
- Kindly fill your ``alt`` parameters in each image.
![Screen Shot 2021-09-28 at 18 35 18](https://user-images.githubusercontent.com/22258274/135179525-92adff2a-62a6-4ec6-8f93-8c88a0e9cd25.png)